common/shlibs

```diff
@@ -3381,3 +3381,38 @@ libcaf_core.so.0.16.2 bro-2.6.1_1
 libcaf_openssl.so.0.16.2 bro-2.6.1_1
 libllhttp.so.1 llhttp-1.0.1_1
 libpinyin.so.13 libpinyin-2.2.1_1
+libgaminggear.so.0 libgaminggear-0.15.1_1
+libgaminggearfx.so.0 libgaminggear-0.15.1_1
+libgaminggearwidget.so.0 libgaminggear-0.15.1_1
+libroccat.so.5 roccat-tools-5.7.0_1
```

Comment:
I don't think there is value in adding such a big amount of libraries from roccat-tools